README.md

```diff
@@ -337,54 +337,54 @@ You can then use the `Slugify::slugify()` method in your controllers:
 $url = Slugify::slugify("welcome to the homepage");
 ```
 
-### Zend Framework 2
+### Laminas
 
-Slugify can be easely used in Zend Framework 2 applications. Included bridge provides a service and a view helper
+Slugify can be easily used in Laminas applications. Included bridge provides a service and a view helper
 already registered for you.
 
 Just enable the module in your configuration like this.
 
 ```php
-return [
+return array(
     //...
 
-    "modules" => [
-        "Application",
-        "ZfcBase",
-        "Cocur\Slugify\Bridge\ZF2", // <- Add this line
+    'modules' => array(
+        'Application',
+        'Cocur\Slugify\Bridge\Laminas' // <- Add this line
         //...
-    ],
+    )
 
     //...
-];
+);
 ```
 
 After that you can retrieve the `Cocur\Slugify\Slugify` service (or the `slugify` alias) and generate a slug.
 
 ```php
-/** @var \Zend\ServiceManager\ServiceManager $sm */
-$slugify = $sm->get("Cocur\Slugify\Slugify");
-$slug = $slugify->slugify("Hällo Wörld");
-$anotherSlug = $slugify->slugify("Hällo Wörld", "_");
+/** @var \Laminas\ServiceManager\ServiceManager $sm */
+$slugify = $sm->get('Cocur\Slugify\Slugify');
```

Comment:
I don't think the service should be used (directly). Either filter or view helper, then it also fits with the integrations in forms and templates.

Reply:
I think if you want to use this in a service or controller, it's helpful.

Reply:
Added an entry for the use of filters